firmware/motor_driver/IGVCMotor.h

```diff
@@ -0,0 +1,38 @@
+#ifndef IGVCMOTORH
+#define IGVCMOTORH
+
+class IGVCMotor {
+    public:
+        IGVCMotor(PinName enable, PinName pwm) {
```

Comment:
Is there any reason we are implementing functions in the header file?

Reply:
For such a short class it seems unnecessary to make two files. It also allows a quick view of all the functionality at once, if needed.